src/Orleans.Runtime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -101,7 +101,7 @@ internal class LocalGrainDirectory :
             Factory<GrainDirectoryPartition> grainDirectoryPartitionFactory,
             RegistrarManager registrarManager,
             ExecutorService executorService,
-            IOptions<GrainBasedMembershipOptions> grainMembershipOptions,
+            IOptions<DevelopmentMembershipOptions> developmentOptions,
```

Comment:
Just to point out that DevelopmentMembership is a new name, which users are not familiar with. If we are going this way, should call it out in the docs or release note.